policy/modules/system/systemd.if

```diff
@@ -246,6 +246,46 @@ interface(`systemd_user_unix_stream_activated_socket',`
 	systemd_user_activated_sock_file($2)
 ')
 
+######################################
+## <summary>
+##   Allow the target domain to be monitored and have its output
+##   captured by the specified systemd user instance domain.
+## </summary>
+## <param name="prefix">
+##	<summary>
+##	Prefix for the user domain.
+##	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	Domain to allow the systemd user instance to monitor.
+##	</summary>
+## </param>
+#
+interface(`systemd_user_app_status',`
```

Comment:
Why does this need to be split out of systemd_user_app_status?

Reply:
This is also a template since it uses prefixes.

Reply:
I decided to split it out since this interface grants a subset of the same access that `systemd_user_daemon_domain()` gives. `systemd_user_daemon_domain()` allows the systemd --user instance to execute the app and monitor it, while `systemd_user_app_status` only allows the systemd --user instance to monitor it.

I don't really see a problem with allowing the exec access though, since the systemd --user instance is one of these domains that we will allow to run other applications like a userdomain, but figured it would allow for more fine-grained access control if needed.

Reply:
Where else would this be called? It's only called one place in this patch. That's the confusion.

Reply:
> Where else would this be called? It's only called one place in this patch. That's the confusion.

This will need to be called by interfaces and templates that grant role or run access to an application, i.e. `screen_role_template()`. I can submit a draft PR for the hooking up of this infrastructure sooner if it would help visualize how this is to be used.

Reply:
I follow now.